src/averageAges.js

```diff
@@ -1,61 +1,86 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+// function to find only men including century specifying
+
+function filterMan(people, sex, century) {
+  const man = people.filter(person => person.sex === sex);
+
+  return century
+    ? man.filter(person => century === Math.ceil(person.died / 100))
+    : man;
+}
+
+// function to define average ages of men
+
+function getManAvg(people, sex, century) {
+  const filtered = filterMan(people, sex, century);
```

Comment:
I guess it's an overkill, this helper function will be used only once, and requires too many arguments